ios/framework/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.4.0</string>
```

Comment:
Is it an issue that there is a trailing zero in the version number here but not in the framework cmake file?